tsl/test/sql/agg_partials_pushdown.sql

```diff
@@ -42,6 +42,10 @@ SELECT count(*), sum(v0), sum(v1), sum(v2), sum(v3) FROM testtable WHERE time >=
 :PREFIX
 SELECT count(*), sum(v0), sum(v1), sum(v2), sum(v3) FROM testtable WHERE time >= '2000-01-01 00:00:00+0'::text::timestamptz AND time <= '2000-02-01 00:00:00+0';
 
+-- Perform chunk append startup chunk exclusion - issue 6282
+:PREFIX
+SELECT count(*), sum(v0), sum(v1), sum(v2), sum(v3) FROM testtable WHERE time >= '2000-01-09 00:00:00+0'::text::timestamptz AND time <= '2000-02-01 00:00:00+0'::text::timestamptz;
```

Comment:
```suggestion
SELECT count(*), sum(v0), sum(v1), sum(v2), sum(v3) FROM testtable WHERE time >= '2000-01-09 00:00:00+0'::timestamptz AND time <= '2000-02-01 00:00:00+0'::timestamptz;
```
Why casting to `text` and then `timestamptz`??

Reply:
The double cast is intentional to get an expression that has to be evaluated at execution time. By just using one cast, we have a constant and this constant can evaluated at planning time. By using the expression, PostgreSQL prefers our `ChunkAppend` node and evaluates the expression during the `Chunks excluded during startup` phase (we have several of these double casts in our current tests to get query plans with a `ChunkAppend` node).